README.md

```
@@ -21,13 +23,18 @@ This app allows you to install and manage applications contained in a single app
 
+## Download
+
+Visit [releases](https://github.com/khcrysalis/Feather/releases) and get the latest `.ipa`.
+
 ## How does it work?
 
 How Feather works is a bit complicated, with having multiple ways to install, app management, tweaks, etc. However, I'll point out how the important features work here.
 
 To start off, we need a validly signed IPA. We can achieve this with Zsign, using a provided IPA using a `.p12` and `.mobileprovision` pair.
 
 #### Install (Server)
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)